### PR TITLE
chore(ext/ffi): Improve FFI static handling

### PR DESCRIPTION
**Pull request by aapoalas**
Minor improvement to FFI static symbols. Mostly just code cleanup.

### CODE REVIEW

File: ext/ffi/lib.rs [hunk 1]
```diff
@@ -1179,76 +1179,89 @@ fn op_ffi_get_static<'scope>(
...
-      serde_v8::from_v8(scope, number.into())?
+      let number: v8::Local<v8::Value> =
+        v8::Number::new(scope, result as f64).into();
+      number.into()
     }
     NativeType::U64 => {
       let result = unsafe { ptr::read_unaligned(data_ptr as *const u64) };
-      let big_int = v8::BigInt::new_from_u64(scope, result);
-      serde_v8::from_v8(scope, big_int.into())?
+      let big_int: v8::Local<v8::Value> =
```
Comment (bartlomieju) [inline ext/ffi/lib.rs:1226]: These type declarations seem to be superfluous, any chance we could remove them?

Comment (aapoalas) [reply ext/ffi/lib.rs:1226]: They're there to facilitate creating the BigInt and immediately calling `.into()`to get the `Local<Value>`, which is in turn needed to call `.into()` for the actual return type `serde_v8::Value`.

File: ext/ffi/lib.rs [hunk 2]
```diff
@@ -1194,76 +1194,89 @@ fn op_ffi_get_static<'scope>(
...
   Ok(match static_type {
     NativeType::Void => {
       return Err(type_error("Invalid FFI static type 'void'"));
     }
     NativeType::U8 => {
       let result = unsafe { ptr::read_unaligned(data_ptr as *const u8) };
-      let number = v8::Number::new(scope, result as f64);
-      serde_v8::from_v8(scope, number.into())?
+      let number: v8::Local<v8::Value> =
+        v8::Number::new(scope, result as f64).into();
```
Comment (piscisaureus) [inline ext/ffi/lib.rs:1206]: Why not use v8::Integer::new() ?

Converting to float and then back to int has a lot of overhead.

Comment (aapoalas) [reply ext/ffi/lib.rs:1206]: Ah, this is a copy-paste error indeed. I think this was what I was originally supposed to fix in this PR (I had a note to self to give statics some love) but forgot all about it.

Comment (aapoalas) [reply ext/ffi/lib.rs]: Massive thanks for noticing this! Leaving this in would've been the height of idiocy and I was completely blind to it.

**Review by littledivy — APPROVED** [review]
LGTM

**Pull request merged**